cindy/index.js

```diff
@@ -0,0 +1,28 @@
+// 36-PB-datastructure-objectscope
+
+// Instructions
+
+//     Create the following object:
+
+// const student = { 
+// firstName: "John", 
+// lastName: "Smith", 
+// class: 12 };
+const student = {
+    firstName: "John",
+    lastName: "Smith",
+    age: 12, 
+    job: "student",
+    class: 12,
+    city: "Paris",
+    getObjectValues: function () {
+      return `${this.firstName} ${this.lastName} is a ${this.age} year old ${this.job} in ${this.city} in class ${this.class} `;
+    },
+  };
+  console.log("student ==> ", student.getObjectValues());
+
```

Comment:
Task 2 ==> You just need to make somehow the same condition like the first task(making an Object)
Task 3 ==> You could use Object.keys() method to find out the length

Reply:
aaaa ok! :v thanks! done it